crates/common/tedge_utils/src/file.rs

```diff
@@ -37,13 +42,13 @@ pub fn create_directory_with_user_group(
     mode: u32,
 ) -> Result<(), FileError> {
     let perm_entry = PermissionEntry::new(Some(user.into()), Some(group.into()), Some(mode));
-    let () = perm_entry.create_directory(Path::new(dir))?;
+    let _ = perm_entry.create_directory(Path::new(dir))?;
     Ok(())
```

Comment:
See https://github.com/thin-edge/thin-edge.io/pull/1208 for a more idiomatic style.